dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -901,6 +901,8 @@ private WorkflowVersion addDockstoreYmlVersionToWorkflow(String repository, Stri
                 workflow.setLastModified(updatedWorkflowVersion.getLastModified());
             }
 
+            // Update verification information.
+            updatedWorkflowVersion.updateVerified();
```

Comment:
Does this method need to be called somewhere for non-dockstore.yml workflows as well?

Reply:
D'oh, I had thought it was a new method that you were invoking here, but the method already exists. 🤦